site/reference/pkg/cat/README.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Cat"
+linkTitle: "cat"
+type: docs
+description: >
+  Print the KRM resources in a file/directory
+---
+
+<!--mdtogo:Short
+    Print the KRM resources in a file/directory
+-->
+
+Cat prints the resources in a package as yaml to stdout.
+
+Cat is useful for printing the KRM resources in a file/directory which might
+contain other non-resource files.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```
+kpt pkg cat [FILE | DIR]
+
+DIR:
+  Path to a directory with KRM resource files. Defaults to the current working directory.
```

Comment:
Descriptions don't match the fact that the argument can be a FILE or DIR. Please do an audit.